core/src/main/scala/quasar/plugin/postgres/datasource/Config.scala

```diff
@@ -21,19 +21,35 @@ import slamdata.Predef._
 import argonaut._, Argonaut._
 
 import java.net.URI
+import java.util.Properties
 
 import quasar.api.datasource.{DatasourceError, DatasourceType}
 import quasar.api.datasource.DatasourceError.InvalidConfiguration
 import quasar.plugin.postgres.datasource.PostgresCodecs._
 
+import org.postgresql.Driver
+
 import scalaz.NonEmptyList
 
 final case class Config(connectionUri: URI, connectionPoolSize: Option[Int]) {
+  import Config._
+
   @SuppressWarnings(Array("org.wartremover.warts.Null"))
   def sanitized: Config = {
     copy(connectionUri = Sanitization.sanitizeURI(connectionUri))
   }
 
+  def properties: Option[Properties] = {
+    val ops = Option(Driver.parseURL(s"jdbc:${connectionUri.toString}", null))
+
+    ops map { ps =>
+      ps.setProperty(
+        DefaultRowFetchSizeKey,
+        ps.getProperty(DefaultRowFetchSizeKey, DefaultRowFetchSize.toString()))
+    }
+
+    ops
```

Comment:
I think you want to remove this so that line 45-49 gets returned?

Reply:
`setProperty` returns `Any`, and in general `ps` is mutable unfortunately